FaceDetection/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8.4)
+
+project(seeta_facedet_lib)
+
+# Build options
+option(BUILD_EXAMPLES  "Set to ON to build examples"  ON)
+option(USE_OPENMP      "Set to ON to build use openmp"  ON)
+
+# Use C++11
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
It seems that the flag "-std=c++11" is not automatically added, and I have to add it manually into `CMAKE_CXX_FLAGS`. The same problem exists for OpenMP. I was wondering why this happens.